installers/osie/main.go

```diff
@@ -82,6 +82,10 @@ func kernelParams(action, state string, j job.Job, s *ipxe.Script) {
 		s.Args("grpc_cert_url=" + grpcCertURL)
 		s.Args("registry_username=" + registryUsername)
 		s.Args("registry_password=" + registryPassword)
+		s.Args("centralized_logging=" + centralized_logging)
```

Comment:
this should not be necessary. The booted machines should be able to figure out if its centralized on its own/may not really care if it configures docker daemon options.

Reply:
`docker logs <service-name>` won't work if there is a log driver defined and eventually this method will throw panic while trying to read the buffer https://github.com/tinkerbell/tink/blob/master/cmd/tink-worker/action.go#L130.

This `centralized_logging` is more of a check if `go captureLogs(ctx, id)`  method should be called or not, https://github.com/tinkerbell/tink/pull/267/files#diff-f776dc055922cd9d519a49a712a9d22fR89